sdk/storage/azure-storage-blob/azure/storage/blob/aio/blob_client_async.py

```diff
@@ -78,6 +78,16 @@ class BlobClient(AsyncStorageAccountHostsMixin, BlobClientBase):  # pylint: disa
         account URL already has a SAS token. The value can be a SAS token string, and account
         shared access key, or an instance of a TokenCredentials class from azure.identity.
         If the URL already has a SAS token, specifying an explicit credential will take priority.
+    :keyword int max_block_size: The maximum upload size used for a block blob. If the
+        block blob exceeds this size, it will be uploaded in chunks. Defaults to 4*1024*1024.
+    :keyword int min_large_block_upload_threshold: The minimum upload size
+        threshold used for a block blob. Defaults to 4*1024*1024+1.
+    :keyword bool use_byte_buffer: Use a byte buffer for block blob uploads. Defaults to False.
+    :keyword int max_page_size: The maximum upload size used for a page blob. If the size of
+        the page blob exceeds this size, it will be uploaded in chunks. Defaults to 4*1024*1024.
+    :keyword int max_single_get_size: The maximum size used for downloading a single blob. Defaults to 32*1024*1024.
+    :keyword int max_chunk_get_size: The maximum chunk size used for downloading a blob. Defaults to 4*1024*1024.
+    :keyword int max_single_put_size: The maximum size used for the upload of a single blob. Defaults to 64*1024*1024.
```

Comment:
I'm hoping @xiafu-msft can comment on whether these descriptions are accurate.
Also whether our default values need updating :)

Reply:
max_single_put_size: if blob size is less than max_single_put_size, then the blob could be uploaded with only one http put request. If blob size is larger than max_single_put_size, the blob will be uploaded in chunks.

Can we put this next to max_block_size? They are related. max_block_size is actually the max chunk size when blob size exceeds max_single_put_size and should be uploaded in chucks